dist/build.gradle

```diff
@@ -26,6 +26,16 @@ ext {
     cliDownloadDir = "${gradle.gradleUserHomeDir}/caches/centraldogma-go/cli"
 }
 
+
+dependencies {
+    // Logging
+    runtimeOnly libs.logback
+    runtimeOnly libs.armeria.logback
```

Comment:
Question) Is `libs.armeria.logback` actually used in this project?
Would it make sense to add this runtime jar if a project actually uses it?

Reply:
It's used with the previous implementation which uses Stringifier. But now we don't need this, so I removed it. 😉